ajax/libs/Bounce/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "bounce",
+  "filename": "bounce.min.js"
```

Comment:
Would you please add `,`(comma) in the end of line?
To fit the json format.
Thank you.